aiodns/__init__.py,sha256=1JcBQOzn9CbOS2Jf09RlxPO20Mqa90uRRRuRBheKXhU,4663
aiodns/error.py,sha256=fnNPLc8ceLk9drumwt-6Un5iiowyWApTe_VIoJ5gT5U,134
aiodns-2.0.0b1.dist-info/LICENSE,sha256=6wRV01EpQl7TmYg81xCSPePiRqUQ4uuE2fAAMtC77Jc,1070
aiodns-2.0.0b1.dist-info/METADATA,sha256=bOVJiAsQhTAE_zMqz5I8cUiRVAuiAkCqhKZgoG-HWfY,3349
aiodns-2.0.0b1.dist-info/WHEEL,sha256=_wJFdOYk7i3xxT8ElOkUJvOdOvfNGbR9g-bf6UQT6sU,110
aiodns-2.0.0b1.dist-info/top_level.txt,sha256=5J2m3NWP4dezFZNpQoHaj8mv472iPcUzFboQoqWcEe4,7
aiodns-2.0.0b1.dist-info/RECORD,,
